.github/workflows/pnp-build-input-api.yml

```diff
@@ -94,9 +108,21 @@ jobs:
         run: |
           dotnet build ${{ secrets.path-to-sln }} --configuration Release /p:Version=${{ steps.semver.outputs.semver }} /p:FileVersion=${{ steps.semver.outputs.version }}
       
+      - name: Create/update Pact broker webhook
+        if:  ${{ inputs.do-pact-provider-tests == true }}
+        uses: extenda/actions/pact-create-webhook@v0
+        with:
+          service-account-key: ${{ secrets.SECRET_AUTH }}
+          pact-application-id: ${{ secrets.pact-application-id }}
+          uuid: 366eebd1-bab9-4632-ac06-0145f8cf29e2
```

Comment:
Can you please add some comment describing what does this hardcoded guid represent?